org.eclipse.xtext.builder.tests/src/org/eclipse/xtext/builder/tests/ParallelBuildEnabledTest.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2018 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.builder.tests;
+
+import static org.junit.Assert.*;
+
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.preferences.DefaultScope;
+import org.eclipse.xtext.builder.internal.Activator;
+import org.eclipse.xtext.builder.preferences.BuilderPreferenceAccess;
+import org.junit.Test;
+
+/**
+ * @author Christian Dietrich - Initial contribution and API
+ */
+public class ParallelBuildEnabledTest {
+
+	@Test
+	public void testIsParallelBuildEnabledInWorkspace() {
+		int maxConcurrentProjectBuilds = DefaultScope.INSTANCE.getNode(ResourcesPlugin.PI_RESOURCES).getInt("maxConcurrentBuilds", -1);
+		assertEquals("parallel build is not enabeld", 8, maxConcurrentProjectBuilds);
```

Comment:
enabled*

Reply:
"was not enabled" might express it better

Reply:
The value is defined in pluginCustomization.ini. There is no way to use a constant. The test here tests if the pluginCustomization.ini has been picked up successfully.

Reply:
public static final String PREF_MAX_CONCURRENT_BUILDS = "maxConcurrentBuilds"; //$NON-NLS-1$

am not sure if this is available in oxygen already

Reply:
an its intended to have a arbitrary number for our tests here

Reply:
I'd expect this to be ignored then. The test will fail then.
We should move to JUnit 5. This will allow us to exclude such tests depending on environment.

Reply:
> an its intended to have a arbitrary number for our tests here

I don't get this comment. What do you mean?

Reply:
for reading it works anyway. i mean if the contant to be used in the test is there

Reply:
well i randonly chose 8 inside the pluginCustomization ini. could be 2,6,5 and 7 as well